### PR TITLE
'lc 16 move whitelisted domains from azure app settings to my sql'

### PR DESCRIPTION
**Pull request by PortobelloRain**
Whitelist removed from Azure to MySQL

### CODE REVIEW

File: src/main/resources/application.yml [hunk 1]
```diff
@@ -1,9 +1,16 @@
...
+    validityInSeconds: ${INVITE_VALIDITY:259200}
+    whitelist:
+      domains: ${WHITELISTED_DOMAINS:abc.com}
+
   data:
     redis:
+
       repositories:
-        enabled: false
+        enabled: falsetomca
```
Comment (m24murray) [inline src/main/resources/application.yml]: typo

File: src/main/resources/application.yml [hunk 2]
```diff
@@ -46,8 +54,8 @@ govNotify:
 invite:
   url: ${INVITE_SIGNUP_URL:http://localhost:8080/signup/%s}
   validityInSeconds: ${INVITE_VALIDITY:259200}
-  whitelist:
-    domains: ${WHITELISTED_DOMAINS:kainos.com, domain.com}
+#  whitelist:
+#    domains: ${WHITELISTED_DOMAINS:kainos.com, domain.com}
```
Comment (m24murray) [inline src/main/resources/application.yml]: remove comment

File: src/main/resources/application.yml [hunk 3]
```diff
@@ -1,9 +1,16 @@
 
 spring:
+  invite:
+    url: ${INVITE_SIGNUP_URL:http://localhost:8080/signup/%s}
+    validityInSeconds: ${INVITE_VALIDITY:259200}
+    whitelist:
+      domains: ${WHITELISTED_DOMAINS:abc.com}
```
Comment (m24murray) [inline src/main/resources/application.yml]: Is this expected?

**Comment by PortobelloRain** [comment]
moved to new PR

**Pull request closed without merging**